packages/server/src/api/controllers/record.js

```diff
@@ -43,30 +44,49 @@ exports.save = async function(ctx) {
   record.type = "record"
   const response = await db.post(record)
   record._rev = response.rev
-  // await ctx.publish(events.recordApi.save.onRecordCreated, {
-  //   record: record,
-  // })
 
+  ctx.eventEmitter.emit(`record:save`, {
+    record,
+    instanceId: ctx.params.instanceId,
+  })
```

Comment:
i think we will want `update` and `create`.

With update, you will be able to pass in `oldRecord` and `newRecord`... meaning you can do something like

`if (oldRecord.status === "Closed" && newRecord.status === "Open") SendEmail( "Re-Opened Email")`

i.e. run things based on **changed** data

Reply:
For sure - would be useful to have those workflow blocks